contracts/UniswapV2Router01.sol

```diff
@@ -9,6 +9,7 @@ import "@nilfoundation/smart-contracts/contracts/Nil.sol";
 import "./interfaces/IUniswapV2Pair.sol";
 
 contract UniswapV2Router01 is IUniswapV2Router01, NilCurrencyBase {
+    // not used now
     address public immutable factory;
```

Comment:
We will probably never use factory in router as it can be on another shard, so let's depricate it completely

Reply:
removed